### ISSUE TITLE
Getting All Reviews - Sorted Newest

### ISSUE THREAD

**Issue by ManuelBanza** [JoMingyu/google-play-scraper#50]
Hi,

I need some help with extracting reviews from an app.

I have 2 problems:

1.  When I try to get the newst reviews from an app it return not all of the newest. For example I can get 3 reviews from yesterday, but when I go to the play store I can see that there were 15 reviews on that day. Also it only returns 100 reviews
**code:**
result= reviews(
    '<app_id>',
    lang='pt', 
    country='pt', 
    sort=Sort.NEWEST, 
    count=10000,
)

result, _ = reviews(
    '<app_id>',
)`

2. I would like to get all of the reviews at once, for example the app that I am trying to scrape has almost 5000 reviews. But when I scrape it, I only get 1500 and they are from 2019 or before that.
code:

`result, continuation_token = reviews(
    '<app_id>',
    lang='pt'
    country='pt'
    sort=Sort.NEWEST
    count=1500
)

result, _ = reviews(
    '<app_id>',
    continuation_token=continuation_token # defaults to None(load from the beginning)
)`

Am I doing something wrong? Call you help? 
Thanks!

**Comment by JoMingyu** [comment]
Can you please give me `app_id` you used?

**Comment by ManuelBanza** [comment]
hi @JoMingyu thank you so much!

I am using portuguese apps, some examples:

- cgd.pt.caixadirectaparticulares
- pt.nos.selfcare
- eu.hboportugal.android

Once again thank you for the help and congrats for this super helpfull library :)

**Comment by tsp2123** [comment]
Hey, I'm having a similar problem. But rather I'm having difficulty understanding what parameters to pass in the continuation_token. Are you supposed to pass the index for the reviews you want to start the scraper from? `continuation_token = 100 ` to start after the 100th review? 

I have also tried:

```
result, continuation_token = reviews(
    'nic.goi.aarogyasetu',
    country='IN', # defaults to 'us'
    count=10, # defaults to 100
)
result, _ = reviews(
    'nic.goi.aarogyasetu',
    country='IN', # defaults to 'us',
    count=10,
    continuation_token=continuation_token
)
```

But get the error, "too many values to unpack"

**Comment by tsp2123** [comment]
![Screen Shot 2020-05-18 at 7 51 49 PM](https://user-images.githubusercontent.com/39416100/82269949-30ddfd00-9941-11ea-8e95-63bcc1d42ed9.png)

**Comment by JoMingyu** [comment]
I am checking. Sorry for being late. I will release the updated version to at least `2020-05-31 20:00:00` KST or comment on the issue.

**Comment by JoMingyu** [comment]
Hi @ManuelBanza .

**Comment by JoMingyu** [comment]
Hi @ManuelBanza .

TL;DR : Non-Issue.

I was able to reproduce the problem in the process of loading a review of an app that corresponds to 'pt.nos.selfcare' among the app_ids I delivered.

Here is what I tested:

1. When the function is called with `count` as 10000, actually 10000 reviews are fetched.
2. After removing duplicates based on `reviewId` in each item of this result, there are 10000 review items.

The following is the code I wrote for testing.

```python
from google_play_scraper import reviews, Sort

for app_id in (
    "cgd.pt.caixadirectaparticulares",
    "pt.nos.selfcare",
    "eu.hboportugal.android",
):
    result, continuation_token = reviews(
        app_id, lang="pt", country="pt", sort=Sort.NEWEST, count=10000,
    )

    assert len(result) == 10000
    assert len(set([r["reviewId"] for r in result])) == 10000
```

When app_id was set to 'pt.nos.selfcare', 'AssertionError' occurred, and the total number of reviews retrieved was 4314.

In fact, if you look at the information of the app in the Play Store, you can see that about 15,000 reviews are reflected in the star rating as shown in the following picture. So I learned about the review function of the Play Store, and realized leaving content in the review function was optional.

<img width="491" alt="스크린샷 2020-05-29 오후 8 19 36" src="https://user-images.githubusercontent.com/21031883/83254958-87252880-a1ea-11ea-8228-1c1dd151f01c.png">

Reviews without content (with stars only) are not shown in the review list. Therefore, it is because only 4314 reviews were loaded, since only the stars were excluded.

Therefore, this problem does not appear to be a library issue. I may have misunderstood your problem, so if so please tag me by attaching the problem code and expected result and actual result to the issue comment.

**Comment by JoMingyu** [comment]
I'll close this issue because it seems to be solved. If any additional problems occurred, reopen this issue or open new issue.
